experiments/AMIP/moist_mpi_earth/coupler_utils/masker.jl

```diff
@@ -0,0 +1,55 @@
+# TODO: build abstraction
+# abstract type AbstractMaskType end
+# struct LandSeaMask{I, FT, D} <: AbstractMaskType
+#     infile:: String
+#     ne:: I 
+#     Nq:: I 
+#     R:: FT 
+#     data:: D
+# end
+
+function LandSeaMask(FT, infile, varname, h_space; outfile = "land_sea_cgll.nc")
+    R = h_space.topology.mesh.domain.radius
+    ne = h_space.topology.mesh.ne
+    Nq = Spaces.Quadratures.polynomial_degree(h_space.quadrature_style) + 1
+
+    mask = ncreader_rll_to_cgll(FT, infile, varname, ne = ne, R = R, Nq = Nq, outfile = outfile)
+    mask = clean_mask.(FT, mask)
+    # LandSeaMask(infile, ne, Nq, R, mask)
+end
+
+"""
+clean_mask(FT, mask)
+- convert to integer values after interpolation (but keep type as floats foe easier calculation (TODO))
+"""
+clean_mask(FT, mask) = mask > FT(0.7) ? FT(1) : FT(0)
```

Comment:
How did we determine the 0.7 threshold?

Reply:
I made it a parameter, not sure how it is chosen